spec/dummy/app/views/layouts/application.html.erb

```diff
@@ -30,7 +30,7 @@
       <header class="app-container-header">
         <span class="app-nav-auth">
           <% if signed_in? %>
-            You are <%= link_to the_current_user, the_current_user %> |
+            You are <%= link_to the_current_user.name, the_current_user %> |
```

Comment:
@glebm I don't think so, this is within the dummy app, so shouldn't really use thredded methods (but can't use to_s because it will blow up...), if you imagine this was a main_app.